doc/source/dev/conduct/enforcement.md

```diff
@@ -0,0 +1,211 @@
+# SciPy Code of Conduct - Enforcement Manual
```

Comment:
Enforcement manual sounds a little scary to me. `Enforcement Procedures`?

Reply:
This is getting repetitious and sounds more like setting up a secret police department. What can we do beyond at most banning someone from the mailing lists and, perhaps, the github repos. Given such limited consequence I think the enforcement machinery can be simplified. More severe consequences start to impinge on the proper role of the court system.